apis/nucleus/src/hooks/useDefaultSelectionActions.js

```diff
@@ -30,8 +30,8 @@ export default function useDefaultSelectionActions({
         onCancel();
         api.cancel();
       },
-      keyboardAction: (e) => {
-        onKeyDeactivate(e);
+      keyboardAction: () => {
+        onKeyDeactivate();
```

Comment:
cleaup: the parameter was not used by the caller of keyboardAction or onKeyDeactivate